azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -493,8 +520,14 @@ public void start() {
         this.executionCleanupIntervalMin, TimeUnit.MINUTES);
     this.cleanupService.scheduleAtFixedRate(this::cleanUpContainersInTerminalStatuses, 0L,
         this.containerCleanupIntervalMin, TimeUnit.MINUTES);
-    this.cleanupService.scheduleAtFixedRate(this::cleanUpDanglingYarnApplications, 0L,
-        this.yarnAppCleanupIntervalMin, TimeUnit.MINUTES);
+    if (this.yarnAppCleanupIntervalMin <= 0) {
```

Comment:
when it is <= 0, it also means the feature is not enabled, right?

Reply:
yes <= 0 is disabled

Reply:
then in this case, your warning message is confused,  "invalid time interval " + this.yarnAppCleanupIntervalMin + ", please correct value "